portal/code/cx-portal-shared-components/src/components/basic/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,20 @@
+import MuiTabs, { TabsProps as MuiTabsProps } from '@mui/material/Tabs'
+import React from 'react'
+
+interface TabsProps extends MuiTabsProps {
+  children?: React.ReactElement[]
+}
+
+export const Tabs = ({ children, ...props }: TabsProps) => {
+  const [value, setValue] = React.useState(0)
```

Comment:
Der State ist hier doppelt, oder? Der wird doch bei der Implementierung in der Parent-Komponente gesetzt und an die `Tabs` übergeben.

```
return (
  <MuiTabs {...props}>
      {children}
  </MuiTabs>
)
```

Reply:
Done